docs_src/OVMS/profileLauncherConfigs.md

```diff
@@ -0,0 +1,63 @@
+
+For the profile launcher, each profile has its own configuration for different pipelines.  The configuration of each profile is done through an yaml configuration file, configuration.yaml.  One example of configuration.yaml is [shown here for classification](https://github.com/intel-retail/automated-self-checkout/blob/main/configs/opencv-ovms/cmd_client/res/classification/configuration.yaml):
+
+```yaml
+OvmsSingleContainer: false
+OvmsServer:
+  ServerDockerScript: start_ovms_server.sh
+  ServerDockerImage: openvino/model_server:2023.1-gpu
+  ServerContainerName: ovms-server
+  ServerConfig: "/models/config.json"
+  StartupMessage: Starting OVMS server
+  InitWaitTime: 10s
+  EnvironmentVariableFiles:
+    - ovms_server.env
+  # StartUpPolicy:
+  # when there is an error on launching ovms server startup, choose one of these values for the behavior of profile-launcher: 
+  #   remove-and-restart: it will remove the existing container with the same container name if any and then restart the container
+  #   exit: it will exit the profile-launcher and 
+  #   ignore: it will ignore the error and continue (this is the default value if not given or none of the above)
+  StartUpPolicy: ignore    
+OvmsClient:
+  DockerLauncher:
+    Script: docker-launcher.sh
+    DockerImage: python-demo:dev
+    ContainerName: classification
+    Volumes:
+      - "$RUN_PATH/results:/tmp/results"
+      - ~/.Xauthority:/home/dlstreamer/.Xauthority
+      - /tmp/.X11-unix
+  PipelineScript: ./classification/python/entrypoint.sh
+  PipelineInputArgs: "" # space delimited like we run the script in command and take those input arguments
+  EnvironmentVariableFiles:
+    - classification.env
+```
+
+The description of each configuration element is explained below:
+
+---
+
+| Configuration Element                   | Description                     |
+| ---------------------------------------------- | ------------------------------------------ |
+| OvmsSingleContainer     | This boolean flag indicates whether this profile is running as a single OpenVino Model Server (OVMS) container or not, e.g. the C-API pipeline use case will use this as `true`. <br>It can indicate the distributed architecture of OVMS client-server when this flag is false. </br>  |
+| OvmsServer                           | This is configuration section for OpenVino Model Server in the case of client-server architecture. |
+| OvmsServer/ServerDockerScript        | The infra-structure shell script to start an instance of OVMS server.  |
+| OvmsServer/ServerDockerImage         | The Docker image tag name for OpenVino Model Server.  |
+| OvmsServer/ServerContainerName       | The Docker container base name for OpenVino Model Server.  |
+| OvmsServer/ServerConfig              | The model config.json file name path for OpenVino Model Server.  |
+| OvmsServer/StartupMessage            | The starting message shown in the console or log when OpenVino Model Server instance is launched. |
```

Comment:
Is this a common message that can be moved into the profile-launcher code?

Reply:
we could, it is now just part of config in the implementation; the refactor is beyond the scope of this documentation PR.

Reply:
it can also be part of common config refactor if we want.

Reply:
I like the idea of adding it to the common config scope.